README.md

```diff
@@ -5,6 +5,36 @@
 This python script helps you build a full [GTK](https://www.gtk.org/) library
 stack for Windows using Visual Studio. Currently, GTK 3 and GTK 4 are supported.
 
+## Install GTK Only
+
+If you want to only run GTK on Windows and not build it yourself, you can download
+a zip file from the last release and unzip it to `C:\gtk`.
+
+It comes with GTK, Cairo, PyGObject, Pycairo, GtkSourceView, adwaita-icon-theme and

```

Comment:
This should probably specify which gtk (3 or 4).

As discussed before the selection here is a bit arbitrary, but if it solves someone's problem I am ok with it

Reply:
I updated the README to add more clarity. I think this is a start, and if others would like us to include other libraries we can in the future.